### ISSUE TITLE
Sign up process tells user "reset your password" and "forgot your password"

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-frontend#5410]
Something changed at one point in the sign up process. Instead of welcoming users who have just signed up, we tell them "reset your password" and direct them to a page "forgot your password".

Expected: 

1. Users who sign up should receive an email: "Hello, thank you for signing up on eventyay.com. Now please verify your email address and set a password for your account. Please click here [link]." The subject should be "Welcome to eventyay.com. Verify your account."

![Screenshot from 2020-10-28 23-14-22](https://user-images.githubusercontent.com/1583873/97502911-5e10eb80-1974-11eb-8292-8e1e7223fff9.png)

2. On the page that users are directed to then please show the following:
* Thank you for verifying your email! Please create a password for your account.
* Instead of "reset password" show "Set Password".
* Do not show "new to the site? create an account..."

![Screenshot from 2020-10-28 23-14-41](https://user-images.githubusercontent.com/1583873/97502893-56514700-1974-11eb-9c44-59fa8787fcd8.png)
3. Ensure the changes do not impact the way passwords are reset for existing accounts. For existing verified accounts, the password reset process should stay as is.

**Comment by iamareebjamal** [comment]
The message linked is not for signup, it is for resetting the password only. And the link is for resetting the password as well.

When users sign up, they get this email:
![image](https://user-images.githubusercontent.com/3874064/97503959-04bcb280-199c-11eb-8388-1bcb088311b7.png)

**Comment by ayush015** [comment]
I would like to try this out . if it is not assigned to anyone

**Comment by mariobehling** [comment]
Ok, I tried the following:
1. with an unregistered email, I tried to get a ticket. 
2. Then message came up "enter your email". I entered the email.
3. I get the message "account is unverified" (for a never used email non existing in the platform)
![Screenshot from 2020-10-29 11-58-31](https://user-images.githubusercontent.com/1583873/97559770-29397e80-19de-11eb-8612-26ae37317b31.png)
4. I get the reset message as described above.
